src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerKeys.scala

```diff
@@ -38,5 +44,12 @@ trait JDKPackagerKeys {
       |    * `icns`: MacOS
       |    * `ico`: Windows
       |    * `png`: Linux
+      |
+      | Defaults to generic Java icon.
     """.stripMargin)
+
+  // To consider:
```

Comment:
Would be interested in knowing if there's already been a debate between an external file vs. explicit settings in SBT. The latter also allows for platform specific scoping.